### DISCUSSION TITLE
Portable App

### DISCUSSION THREAD

**Discussion by usarmyjoe** [simonec73/threatsmanager#32] [Ideas]
I love this project. Thank you! I only wish I could share it with my peers more easily. Being good security folks, our machines do not allow installations of unapproved software. I do not know if it is possible, but a portable app would be fantastic for this purpose.

**Comment by simonec73** [comment]
> I love this project. Thank you! I only wish I could share it with my peers more easily. Being good security folks, our machines do not allow installations of unapproved software. I do not know if it is possible, but a portable app would be fantastic for this purpose.

Thank you for your comment. Yes, that may be possible, but there is a dependency I need to take care of. The integrated help system uses some libraries from Microsoft to integrate Edge. I will need to make it optional. I will work on this for next version. Thank you!

**Reply by simonec73** [reply]
Consider though that it may be more secure to install a tool in the appropriate folders, than using the same tool as a portable one, because of the embedded OS protections. Therefore the recommended approach will still be to use the current installation method.

**Comment by simonec73** [comment]
TMS has now a portable edition, which is available since version 2.1.1.
